### PR TITLE
Upgrade nebula-go

### PR DESCRIPTION
**Pull request by jievince**

### CODE REVIEW

File: printer/plan_desc_printer.go [hunk 1]
```diff
@@ -148,7 +154,8 @@ func (p PlanDescPrinter) renderDotGraphByStruct() string {
 	}
 	builder.WriteString("}")
 	p.writer.AppendRow(table.Row{builder.String()})
-	return p.writer.Render()
+	fmt.Println(p.writer.Render())
```
Comment (yixinglu) [inline printer/plan_desc_printer.go:157]: don't dump this strings to console in render function

File: printer/plan_desc_printer.go [hunk 2]
```diff
@@ -223,10 +233,14 @@ func (p PlanDescPrinter) renderDotGraph() string {
...
```
Comment (yixinglu) [inline printer/plan_desc_printer.go:236]: ditto

File: printer/plan_desc_printer.go [hunk 3]
```diff
@@ -282,5 +296,11 @@ func (p PlanDescPrinter) renderByRow() string {
 		row = append(row, strings.Join(columnInfo, "\n"))
 		p.writer.AppendRow(table.Row(row))
 	}
-	return p.writer.Render()
+	fmt.Println(p.writer.Render())
+	return "renderByRow"
```
Comment (yixinglu) [inline printer/plan_desc_printer.go:300]: return rendered string rather than print

File: printer/dataset_printer.go [hunk 4]
```diff
@@ -14,7 +14,7 @@ import (
 	"strings"
 
 	"github.com/jedib0t/go-pretty/v6/table"
-	"github.com/vesoft-inc/nebula-go/v2/nebula"
+	nebula0 "github.com/vesoft-inc/nebula-clients/go/nebula"
```
Comment (yixinglu) [inline printer/dataset_printer.go:17]: Why named `nebula0`?

Comment (jievince) [reply printer/dataset_printer.go:17]: Because there is another package named 'nebula'

**Review by Aiee — DISMISSED** [review]

**Review by Aiee — APPROVED** [review]

**Pull request merged**